examples/source/interactivity-dynamic-content/Tictactoe.html

```diff
@@ -0,0 +1,319 @@
+<!---
+
+preview: default
+teaserImage: '/static/samples/img/teaser/tictactoe.png'
+author: garanj
+
+--->
+
+<!-- ## Introduction -->
+<!--
+This is a sample showing how to implement Tic-Tac-Toe. This includes:
+
+*   How to use `amp-state` to maintain the state of play.
```

Comment:
Please indent all three list items as they're not recognised as a list otherwise.

Reply:
Done!

Reply:
Please indent line 13, 14 and 15. They were the ones I initially meant. Thanks! 😃

Reply:
Ah apologies! Hopefully fixed now.